### PR TITLE
BaSP-MR-123: Delete functionality BUG

### PR DESCRIPTION
**Pull request by IvanJukonis**
The current PR is related to ticket 123. To resolve it, the TRAINERS component needs to have the DELETE functionality implemented correctly.

All the requirements listed in the ticket have been completed.

Problem: The DELETE function uses the ID stored in the TRAINERS state. When a trainer is added, the generated ID of the response is not saved in the TRAINERS state. Therefore, when performing the ADD action without refreshing the page, all trainers stored in the TRAINERS state do not have an ID, which makes the DELETE functionality impossible to use.

Solution: Every time an ADD action is performed on trainers, the ID is obtained from the response, and the complete trainer is stored within the TRAINERS state.

### CODE REVIEW

**Comment by IvanJukonis** [comment]
[[BUG] -Trainer - Error Delete](https://trello.com/c/76HQ5YNF/123-bug-trainer-error-delete)

File: src/Components/Trainers/TableTrainers/index.jsx [hunk 1]
```diff
@@ -31,6 +31,7 @@ const Table = ({
   };
 
   const somefunction = (id) => {
+    console.log(id);
```
Comment (maurojjzz) [inline src/Components/Trainers/TableTrainers/index.jsx:34]: delete this console.log

File: src/Components/Trainers/FormTrainers/index.jsx [hunk 2]
```diff
@@ -67,45 +66,42 @@ const Form = ({ addItem, closeForm }) => {
...
       ...trainer,
       [e.target.name]: e.target.value
     });
   };
 
-  const onSubmit = (e) => {
+  const onSubmit = async (e) => {
     e.preventDefault();
     setModalConfirmAdd(false);
     console.log(validateForm());
```
Comment (maurojjzz) [inline src/Components/Trainers/FormTrainers/index.jsx:79]: delete this console.log

**Review by maurojjzz — CHANGES_REQUESTED** [review]

**Review by maurojjzz — APPROVED** [review]
nice work

**Pull request merged**